bindings/nodejs/examples/how_tos/alias/governance-transition.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    AliasOutput,
+    StateControllerAddressUnlockCondition,
+    UnlockConditionType,
+    Utils,
+    Wallet,
+    initLogger,
+} from '@iota/sdk';
+
+// This example uses secrets in environment variables for simplicity which should not be done in production.
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./how_tos/alias/governance-transition.ts
+
+// In this example we will update the state controller of an alias output.
+async function run() {
+    initLogger();
+    if (!process.env.FAUCET_URL) {
+        throw new Error('.env FAUCET_URL is undefined, see .env.example');
+    }
+    if (!process.env.WALLET_DB_PATH) {
+        throw new Error('.env WALLET_DB_PATH is undefined, see .env.example');
+    }
+    if (!process.env.STRONGHOLD_PASSWORD) {
+        throw new Error(
+            '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
+        );
+    }
+    try {
+        // Create the wallet
+        const wallet = new Wallet({
+            storagePath: process.env.WALLET_DB_PATH,
+        });
+
+        // Get the account we generated with `01-create-wallet`
+        const account = await wallet.getAccount('Alice');
+
+        // May want to ensure the account is synced before sending a transaction.
+        const balance = await account.sync();
+
+        if (balance.aliases.length == 0) {
+            throw new Error(`No Alias available in account 'Alice'`);
+        }
+
+        // We try to update the first alias in the account
+        const aliasId = balance.aliases[0];
+
+        const aliasOutputData = (
+            await account.unspentOutputs({ aliasIds: [aliasId] })
+        )[0];
+        console.log(
+            `Alias ${aliasId} found in unspent output: ${aliasOutputData.outputId}`,
+        );
+
+        await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+
+        const newStateController = Utils.parseBech32Address(
```

Comment:
Do we really need to call parse here?

Reply:
Yes because it's a bech32 address string, but we need to provide is as address with type

Reply:
Why do we not already deserialize the address as the proper type from the generate address call?

Reply:
I think I misunderstood 🤷‍♀️